src/x/instrument/config.go

```diff
@@ -60,11 +60,15 @@ type AttributionConfiguration struct {
 	// Sampling rate
 	SamplingRate float64 `yaml:"samplingRate" validate:"nonzero,min=0.0,max=1.0"`
 
-	// Matched labels of this attribution
+	// Matched labels of this attribution, if a time series has label A & B & C, and here we attribute
+	// to A, we will count all ts with coordinator_attribution_A_sample_count{A="value for label A"}
 	Labels []string `yaml:"labels"`
 
-	// Filter metrics for attribution
-	Filters []string `yaml:"filters"`
```

Comment:
We have existing yaml configuration files with this deleted field. I suppose that won't cause parse failures.

Reply:
I will update config with new docker images when check in, so it won't cause issue.